sns_ik_lib/matlab/TEST_vel_ik_solvers.m

```diff
@@ -0,0 +1,39 @@
+% Run a simple unit test on the matlab implementation for the various
```

Comment:
Unless it affects how matlab builds documentation (?), i would move the description below the Copyright and License

Reply:
Matlab uses the first comment block after a function definition as the documentation that is auto-generated by the help command. I looked at how the license is included in the Matlab source code, and modified our license to be formatted similarly: leave one blank line after the help file and then include the license as a comment.